OPHD/States/MapViewState.cpp

```diff
@@ -421,22 +421,22 @@ void MapViewState::onKeyDown(NAS2D::EventHandler::KeyCode key, NAS2D::EventHandl
 
 		case NAS2D::EventHandler::KeyCode::KEY_w:
 		case NAS2D::EventHandler::KeyCode::KEY_UP:
-			mMapView->moveView(MapOffset{DirectionNorthWest * shiftMoveScalar, 0});
+			mMapView->moveView(MapOffset {DirectionNorthWest * shiftMoveScalar});
 			break;
 
 		case NAS2D::EventHandler::KeyCode::KEY_s:
 		case NAS2D::EventHandler::KeyCode::KEY_DOWN:
-			mMapView->moveView(MapOffset{DirectionSouthEast * shiftMoveScalar, 0});
+			mMapView->moveView(MapOffset{DirectionSouthEast * shiftMoveScalar});
 			break;
 
 		case NAS2D::EventHandler::KeyCode::KEY_a:
 		case NAS2D::EventHandler::KeyCode::KEY_LEFT:
-			mMapView->moveView(MapOffset{DirectionSouthWest * shiftMoveScalar, 0});
+			mMapView->moveView(MapOffset{DirectionSouthWest * shiftMoveScalar});
 			break;
 
 		case NAS2D::EventHandler::KeyCode::KEY_d:
 		case NAS2D::EventHandler::KeyCode::KEY_RIGHT:
-			mMapView->moveView(MapOffset{DirectionNorthEast * shiftMoveScalar, 0});
+			mMapView->moveView(MapOffset{DirectionNorthEast * shiftMoveScalar});
```

Comment:
This doesn't appear to be using the new `operator*`. It's still using the existing `operator*` on the `NAS2D::Vector<int>` class. The only change is there is no explicit initialization value for `z`. Since the `struct` is initialized with `{}`, unspecified fields should be default initialized, which in this case should result in a `0` for `z`.

I assume what you meant to write was:
```cpp
mMapView->moveView(MapOffset{DirectionNorthEast, 0} * shiftMoveScalar);
```

Of course with the new `operator*` that means we can go back to using the predefined `MapOffset` values from `DirectionOffset.h`:
```cpp
mMapView->moveView(MapOffsetNorthEast * shiftMoveScalar);
```